.travis/tests.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+set -e
+set -x
+
+source .venv/bin/activate
+
+python3 ./tests/runAllTests.py
```

Comment:
make sure this script returns non-zero exit code on failure, otherwise Travis CI doesn't see if a test failed